.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Documentation & Website
+about: Issues regarding documentation and website
+title: '[DOC/WEBSITE] '
+labels: Doc, Website, needs triage
```

Comment:
these two issues are related but not the same. maybe either split into two or pick one? otherwise the labels for doc and website will have decreased SNR